app/test_split_integer.py

```diff
@@ -2,20 +2,21 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert set(split_integer(16, 4)) == {4}
```

Comment:
This solution is correct, I'm just thinking will it be as descriptive as writing the result directly? 🤔 What do you think?